app/src/main/java/org/oppia/app/player/state/StateFragmentPresenter.kt

```diff
@@ -354,6 +357,46 @@ class StateFragmentPresenter @Inject constructor(
     itemList.add(contentViewModel)
     stateAdapter.notifyDataSetChanged()
   }
+  private fun checkAndAddInteraction() {
```

Comment:
Perhaps instead: addInteractionForPendingState(). That indicates what's being checked.

Reply:
done.